service/view_building_coordinator.cc

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright (C) 2025-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+ */
+
+#include <exception>
+#include <iterator>
+#include <ranges>
+#include <seastar/core/coroutine.hh>
+
+#include "cql3/query_processor.hh"
+#include "db/schema_tables.hh"
+#include "db/system_keyspace.hh"
+#include "mutation/canonical_mutation.hh"
+#include "schema/schema_fwd.hh"
+#include "seastar/core/loop.hh"
```

Comment:
Seastar includes use angle brackets.

Also, why do you need loop constructs? With coroutines they are unnecessary.

Reply:
It has to be some auto include, removed.